specs/commandline/policy.md

```diff
@@ -0,0 +1,696 @@
+# notation policy
+
+## Description
+
+Use `notation policy` command to add/update/show/delete trust policies. As part of signature verification workflow, user needs to configure the trust policies to specify trusted identities that sign the artifacts, and the level of signature verification to use. For more details, see [trust policy spec](https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md#trust-policy).
+
+An example of `trustpolicy.json`:
+
+```json
+{
+    "version": "1.0",                                                   // version info
+    "trustPolicies": [                                                  // list of trust policy statements
+        {
+            "name": "wabbit-networks-dev",                              // Name of the first policy statement
+            "registryScopes": [ "dev.wabbitnetworks.io/net-monitor" ],  // The registry artifacts to which the policy applies
+            "signatureVerification": {                                  // The level of verification - strict, permissive, audit, skip
+                "level": "strict"
+                "override" : {
+                     "expiry" : "log",
+                     "authenticity": "log"
+                }
+            },
+            "trustStores": [ "ca:wabbit-networks-dev" ],                // The trust stores that contains the X.509 certificates
+            "trustedIdentities": [                                      // Identities that are trusted to sign the artifact.
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io"
+            ]
+        },
+        {
+            "name": "wabbit-networks-prod",                             // Name of the second policy statement.
+            "registryScopes": [ "prod.wabbitnetworks.io/net-monitor" ],       
+            "signatureVerification": {                                
+                "level": "permissive"
+            },
+            "trustStores": [ "ca:wabbit-networks-prod" ],                  
+            "trustedIdentities": [                                    
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Prod, CN=wabbit-networks.io"
+            ]
+        }
+    ]
+}
+```
+
+## Outline
+
+### notation policy command
+
+```text
+Manage trust policies for signature verification.
+
+Usage:
+  notation policy [command]
+
+Available Commands:
```

Comment:
We also need a mechanism to export this trust policy to a file so that users can share/deploy them later in the systems like ratify etc.

This can be achieved by either displaying trust policy location or adding an `export` command or adding a flag `--export` in the list command,

Reply:
@priteshbandi Maybe we can reuse `notation policy show`, currently it is called `notation policy list`, maybe `show` is better, so that, by default, the content of trust policy file is displayed. Does it make sense?